packages/neos-ui/src/Sagas/CR/NodeOperations/helpers.js

```diff
@@ -49,6 +49,7 @@ export const calculateDomAddressesFromMode = (mode, contextPath, fusionPath) =>
             const element = findNodeInGuestFrame(contextPath, fusionPath);
 
             return {
+                parentContextPath: contextPath,
```

Comment:
Naming here is not cool. The method name says `calculateDomAddressesFromMode` and the actual returned keys mention "parent". Change either one or another.